source/test/default_mode_test/SymWorld.test.cc

```diff
@@ -458,61 +382,34 @@ TEST_CASE( "SymDoBirth", "[default]" ) {
     WHEN( "free living symbionts are allowed"){
       config.FREE_LIVING_SYMS(1);
       w.SetFreeLivingSyms(true);
-      config.SYM_LIMIT(3);
-
-      emp::Ptr<Organism> host1 = new Host(&random, &w, &config, int_val);
-
-      emp::Ptr<Organism> sym1 = new Symbiont(&random, &w, &config, int_val);
-      emp::Ptr<Organism> sym2 = new Symbiont(&random, &w, &config, int_val);
 
-      WHEN("sym is inserted into an empty world"){
```

Comment:
This seems odd to delete, an also odd since the THEN seems to rely on this section and refers to "it"